python/README.md

```diff
@@ -0,0 +1,32 @@
+# Python
+
+> There are 2 alternative options availible:
+>
+> - [using deltabot](../python_deltabot_plugin) (bot framework, includes features as chat command parsing)
+> - [using simplebot](../python_simplebot_plugin) (simplebot is a maintained fork of deltabot with many plugins availible)
+
+## Instalation
+
+```sh
+# Optional create virtualenv
+pip install virtualenv
```

Comment:
```suggestion
pip3 install virtualenv
```

When I tried to run that on xubuntu, I got: `Command 'pip' not found, but there are 18 similar ones.`. with pip3 it worked without problems, and as we are using pip3 anyway 3 lines below, let's roll with that?